plugins/linux-capture/xcompcap-main.cpp

```diff
@@ -621,6 +621,7 @@ void XCompcapMain::tick(float seconds)
 	}
 
 	XDisplayLock xlock;
+	XErrorLock xerrlock;
```

Comment:
we are already holding an xserver lock in XDisplayLock, did you mean to replace this with the XErrorLock?

Reply:
Yes, I didn't realize a XErrorLock supersedes a XDisplayLock. Thanks!